src/native_mux_impl/stdio.rs

```diff
@@ -68,10 +68,29 @@ impl Fd {
         }
     }
 
-    fn new_owned<T: IntoRawFd>(fd: T) -> Self {
+    unsafe fn new_owned<T: IntoRawFd>(fd: T) -> Self {
```

Comment:
We should document the safety requirements of `new_owned` here now that it's `unsafe`.

Reply:
Done!